AWSAuthSDK/Tests/AWSMobileClientTests/AWSMobileClientTestBase.swift

```diff
@@ -224,7 +224,11 @@ class AWSMobileClientTestBase: XCTestCase {
         let keychain = AWSUICKeyChainStore(service: "\(bundleID!).\(AWSCognitoIdentityUserPool.self)")
         let namespace = "\(AWSMobileClient.default().userPoolClient!.userPoolConfiguration.clientId).\(username)"
         let key = "\(namespace).tokenExpiration"
-        keychain.removeItem(forKey: key)
+        
```

Comment:
I think this invalidation is different, from the existing one. The existing invalidation removes the expiration date so that it falls to the refresh token expire flow. I think what you are trying to do is accesstoken expiry flow. 

Can we split the above method into two:

```
func mockRefreshTokenExpired() -> same as existing code

func mockAccessTokenExpired() -> your change as per this PR

```

Reply:
Ah, you're right, and I should have dug into this more deeply before making the change. Will update.